### DISCUSSION TITLE
I'm doing it... progress

### DISCUSSION THREAD

**Discussion by vjoco** [OpenGammaProject/Open-Gamma-Detector#62] [Show and tell]
Board ordered at pcbway, resized for 2 layers
### Part 1, 30V power supply, approx 1h work
- soldering component
- supply 5V on VSYS
- get 30 V at the output, nice

![image](https://github.com/user-attachments/assets/a6ce2979-b13a-44b4-9ace-d71362af3710)

**Comment by NuclearPhoenixx** [comment]
Neat, good luck for the rest of the soldering!

**Comment by vjoco** [comment]
It's done.
 I was using a GSO crystal from ebay, 6x4x20mm. The number of ph/MeV is 20% of a NaI so I boost the amplification stage. I had to modify the discriminator because the offset increased.

 This is the first pulse I got:
![image](https://github.com/user-attachments/assets/b11d0f0d-7cdb-482e-94a5-c71788f805d9)

 I could find 60% KCl at a supermarket, this is a photo of the setup:
![image](https://github.com/user-attachments/assets/ccbb1307-60ef-44b1-9db1-4a2420c59b37)

And this is the first spectra (10 minutes aquisition):
![image](https://github.com/user-attachments/assets/d92c63b2-e33b-4e09-b31a-b76368046f91)

Very nice project!

I'm leaving now for a 10h acquisition. K should be there :)

**Reply by NuclearPhoenixx** [reply]
Hey there, glad you're making progress! Pulses might be a bit too fast if you're encountering any problems. Looks like you should either turn up the gain a little bit or tune down the threshold. It shouldn't cut off that quickly, happy hacking!

**Reply by vjoco** [reply]
Thanks, pulses are fast probably because iI use a GSO-Ce. I had to boost the gain and make the SH circuit faster.

**Comment by vjoco** [comment]
In the end everything works very nice, I share some photos/results:

![pic_setup](https://github.com/user-attachments/assets/c3c8099c-a1ff-43c1-b3b5-fcb96ec4ac1e)

![04477xoOE8TVyYob](https://github.com/user-attachments/assets/2d0c844e-9b0c-4556-9918-8d29136c13b0)

![10Th](https://github.com/user-attachments/assets/f314402e-1da4-43df-aae2-aac8c519d090)

![sample_and_hold](https://github.com/user-attachments/assets/8f768bee-d9d3-4f90-8df9-0e30a21d309c)
